src/main/java/graphql/schema/idl/DefaultSchemaPrinterComparatorRegistry.java

```diff
@@ -0,0 +1,93 @@
+package graphql.schema.idl;
+
+import graphql.schema.GraphQLType;
+
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.UnaryOperator;
+
+import static graphql.Assert.assertNotNull;
+
```

Comment:
Thinds that we want to be true public API we annotate for `@PublicApI`.  See other class for examples